### PR TITLE
STY: update standard based on ecosystem

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses #45

Updates document standards

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

inspection of docs

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
- [x] Update zenodo.json file for new code contributors

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,43 +5,43 @@ Bug reports, feature suggestions and other contributions are greatly
...
+
 * Detailed steps to reproduce the bug
 
-Feature requests
-^^^^^^^^^^^^^^^^
+Feature requests and feedback
+-----------------------------
+
+The best way to send feedback is to file an issue at
+[GitHub](https://github.com/pysat/pysatNASA/issues).
```
Comment (JonathonMSmith) [inline CONTRIBUTING.md:37]:
```suggestion
[GitHub](https://github.com/pysat/pysatCDAAC/issues).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -50,30 +50,49 @@ More about Development
...
-   should be named ``pysatCDAAC/tests/test_methods_usercode.py``.  If no test
-   file exists, then you should create one.  This testing uses pytest, which
-   will run tests on any python file in the test directory that starts with
-   ``test``.  Test classes must begin with ``Test``, and test methods must also
-   begin with ``test``.
-
-5. When you're done making changes, run all the checks to ensure that nothing
+  Tests for custom functions should be added to the appropriately named file
+  in ``pysatNASA/tests``.  For example, custom functions for the OMNI HRO data
+  are tested in ``pysatNASA/tests/test_omni_hro.py``.  If no test file exists,
```
Comment (jklenzing) [inline CONTRIBUTING.md:75]:
```suggestion
  in ``pysatCDAAC/tests``.  For example, custom instrument functions are
  tested in ``pysatCDAAC/tests/test_instruments.py``.  If no test file exists,
```

File: .github/workflows/pip_rc_install.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # This workflow will install Python dependencies and the latest RC of pysatNASA from test pypi.
```
Comment (jklenzing) [inline .github/workflows/pip_rc_install.yml:1]:
```suggestion
# This workflow will install Python dependencies and the latest RC of pysatCDAAC from test pypi.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -50,30 +50,49 @@ More about Development
...
-   methods contained in ``pysatCDAAC/instruments/methods/usercode.py``
-   should be named ``pysatCDAAC/tests/test_methods_usercode.py``.  If no test
-   file exists, then you should create one.  This testing uses pytest, which
-   will run tests on any python file in the test directory that starts with
-   ``test``.  Test classes must begin with ``Test``, and test methods must also
-   begin with ``test``.
-
-5. When you're done making changes, run all the checks to ensure that nothing
+  Tests for custom functions should be added to the appropriately named file
+  in ``pysatNASA/tests``.  For example, custom functions for the OMNI HRO data
```
[the remark this answers is not in the document]
Comment (jklenzing) [reply CONTRIBUTING.md]: good catch. There are some custom tests there, so I pointed to it anyway

**Review by JonathonMSmith — APPROVED** [review]

**Pull request merged**